tutorials/pulsar_analysis.ipynb

```diff
@@ -0,0 +1,464 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Making an angular cut to retain only the events within 0.2 deg\n",
+    "on_radius = 0.2 * u.deg\n",
+    "one_obs_vela_select = obs_list_vela[0].events.select_sky_cone(center=pos_target, radius=on_radius)\n",
+    "\n",
+    "print(one_obs_vela_select)\n",
+    "\n",
+    "print(\"\\nWe can access the first 20 MJD this way:\\n\")\n",
+    "print(one_obs_vela_select.time[:20])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## II : Phaseograms"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once you have phases, here is how you can make a phasogram for each observation:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# A phasogram is a histogram so you can set the binning the same way:\n",
+    "\n",
+    "nbins = 30  # Number of bins\n",
+    "lim_phaso = (0, 1)  # Range of phaseogram\n",
+    "size_bin = (lim_phaso[1] - lim_phaso[0])/nbins  # Size of each bin\n",
+    "midbins = np.linspace(lim_phaso[0] + 0.5 * size_bin, lim_phaso[1] - 0.5 * size_bin, nbins)  # Position of each middle of bin"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "phases = one_obs_vela_select.table['PHASE'].data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "values = np.histogram(phases, range = lim_phaso, bins=nbins)[0]\n",
+    "# Uncertainty on each bin\n",
+    "count_err = values ** 0.5\n",
+    "plt.bar(x = midbins, height = values, width = size_bin, color='#d53d12', alpha = 0.8, edgecolor = 'black', yerr=count_err)\n",
+    "plt.xlim(0, 1)\n",
+    "plt.title('Phaseogram with angular cut of {}'.format(on_radius))\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's add some fancy additions to our phasogram !"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Let's redo the same plot for the basis\n",
+    "plt.bar(x = midbins, height = values, width = size_bin, color='#d53d12', alpha = 0.8, edgecolor = 'black', yerr=count_err)\n",
+    "plt.xlim(0, 1)\n",
+    "plt.title('Phaseogram with angular cut of {}'.format(on_radius))\n",
+    "\n",
+    "# Evaluate background level\n",
+    "mask_off = (0.7 < phases) & (phases < 1)  # mask for the off-phase zone\n",
+    "count_bkg = mask_off.sum()  # number of phases in the off-phase zone\n",
+    "bkg = count_bkg / nbins / 0.3  # bkg lebel normalized by the size of the OFF zone (0.3)\n",
+    "\n",
+    "# Plot background level\n",
+    "x_bkg = np.linspace(0, 1, 50)\n",
+    "plt.plot(x_bkg, bkg*np.ones(len(x_bkg)), color='black', alpha=.5, ls='--', lw=2)\n",
```

Comment:
Since you have an error on the bkg estimate, you might use a band rather than a horizontal line?